### PR TITLE
Add radio input to admin panel

### PR DESCRIPTION
**Pull request by IvanBuljovcic**
Admin panel:

* Toggle radio switch
* If switch is _on_ show radio station input

### CODE REVIEW

**Comment by IvanBuljovcic** [comment]
https://trello.com/c/v2JeOYqs/599-add-radio-editing-to-the-admin

File: src/api/admin.ts [hunk 1]
```diff
@@ -231,6 +233,7 @@ export const createVenue = async (input: VenueInput, user: UserInfo) => {
 };
 
 export const updateVenue = async (input: VenueInput, user: UserInfo) => {
+  console.log("input: ", input);
```
Comment (thecadams) [inline src/api/admin.ts:236]: nit: unnecessary console.log

File: functions/venue.js [hunk 2]
```diff
@@ -81,11 +81,13 @@ const createVenueData = (data, context) => {
     placement: { ...data.placement, state: PlacementState.SelfPlaced },
     showLiveSchedule: data.showLiveSchedule ? data.showLiveSchedule : false,
     showChat: true,
-    showRangers: data.showRangers || false,
+    showRangers: data.showRangers,
```
Comment (thecadams) [inline functions/venue.js:84]: curious: why did the `|| false` go away here?

File: functions/venue.js [hunk 3]
```diff
@@ -81,11 +81,13 @@ const createVenueData = (data, context) => {
...
     parentId: data.parentId,
     attendeesTitle: data.attendeesTitle || "partygoers",
     chatTitle: data.chatTitle || "Party",
-    requiresDateOfBirth: data.requiresDateOfBirth || false,
+    requiresDateOfBirth: data.requiresDateOfBirth,
```
Comment (thecadams) [inline functions/venue.js:88]: curious: as above re `|| false` disappearing

File: functions/venue.js [hunk 4]
```diff
@@ -81,11 +81,13 @@ const createVenueData = (data, context) => {
...
+    showRadio: data.showRadio,
+    radioStations: [data.radioStations] || [],
```
Comment (thecadams) [inline functions/venue.js:90]:
```suggestion
    radioStations: data.radioStations ? [data.radioStations] : [],
```

File: src/pages/Admin/Venue/DetailsForm.tsx [hunk 5]
```diff
@@ -767,6 +767,33 @@ const DetailsFormLeft: React.FC<DetailsFormLeftProps> = ({
...
+  const renderRadioStationInput = () => (
+    <div className="input-container">
+      <h4 className="italic input-header">Radio station (stream):</h4>
+      <input
+        type="text"
+        disabled={disable}
+        name={`radioStations`}
+        ref={register}
+        className="wide-input-block input-centered align-left"
+        placeholder="Radio station"
```
Comment (thecadams) [inline src/pages/Admin/Venue/DetailsForm.tsx:789]:
```suggestion
        placeholder="Radio station URL..."
```

File: src/pages/Admin/Venue/DetailsForm.tsx [hunk 6]
```diff
@@ -767,6 +767,33 @@ const DetailsFormLeft: React.FC<DetailsFormLeftProps> = ({
...
+      <label id="showRadio" className="switch">
+        <input type="checkbox" id="showRadio" name="showRadio" ref={register} />
+        <span className="slider round" />
+      </label>
+    </div>
+  );
+
+  const renderRadioStationInput = () => (
+    <div className="input-container">
+      <h4 className="italic input-header">Radio station (stream):</h4>
```
Comment (thecadams) [inline src/pages/Admin/Venue/DetailsForm.tsx:782]:
```suggestion
      <h4 className="italic input-header">Radio station stream URL:</h4>
```

**Review by thecadams — COMMENTED** [review]
Looking good so far, couple of comments, thanks for this one!

File: functions/venue.js (hunk 2 above)
Comment (thecadams) [inline functions/venue.js:84]:
```suggestion
    showRangers: data.showRangers ?? false,
```

File: functions/venue.js (hunk 3 above)
Comment (thecadams) [inline functions/venue.js:88]:
```suggestion
    requiresDateOfBirth: data.requiresDateOfBirth ?? false,
```

**Review by thecadams — COMMENTED** [review]
Suggesting using null-coalescing operators for these

**Review by thecadams — APPROVED** [review]
lgtm!

**Review by denisdimitrov — CHANGES_REQUESTED** [review]

File: functions/venue.js [hunk 7]
```diff
@@ -81,11 +81,13 @@ const createVenueData = (data, context) => {
     placement: { ...data.placement, state: PlacementState.SelfPlaced },
     showLiveSchedule: data.showLiveSchedule ? data.showLiveSchedule : false,
     showChat: true,
-    showRangers: data.showRangers || false,
+    showRangers: data.showRangers ?? false,
```
[the remark this answers is not in the document]
Comment (denisdimitrov) [reply functions/venue.js:84]: You can see the error in the terminal once you run `firebase:emulate-functions`

**Pull request merged**